### ISSUE TITLE
Add a screenshot

### ISSUE THREAD

**Issue by stephanenicolas** [hcoles/highwheel#7]
Could.you add an example of analysis to help decide if this plugin matches devs needs ? 

I am android dev and I am thinking of working ton a gradle plugin o make your tool work on Android.
Before doing so, it would be nice to see an example of the output of high wheel.

Thx in advance !

**Comment by szpak** [comment]
A screenshot or an analysis report for a sample FOSS project would be useful to have tangible result at a glance.

**Comment by CodeingBoy** [comment]
Agree. Also, please consider adding output location(`\target\highwheel`). I got confused because it generates no output at project directory, until I found output directory after 15 minutes.
